### ISSUE TITLE
Support for local configuration

### ISSUE THREAD

**Issue by GHNewbiee** [commenthol/md-fileserver#10]
Please consider to add support for local configuration in any of the following ways:
- by default use of `config.js` when the last is found in the root directory
- by adding command option such as `-C, --config` in which, either the directory in which the `config.js` lies (e.g. `./config/md/`), or the full path of the config file (e.g. `./config/md/OTHER-NAME-config.js`), is given.

IMHO, global config file through *personalisation* may restrict the parallel running of different projects and is unfair for the app itself.

**Comment by commenthol** [comment]
Hi @GHNewbiee 
Did you already try `mdstart /config` ?
Maybe that's what you are looking for?

**Comment by GHNewbiee** [comment]
Hi @commenthol 

> Did you already try `mdstart /config` ?

Yes, I did. What I have observed is that, after submitting,
- a local `config.js` is not created at the root directory. (I have already given permission, through exception, session cookies to be accepted from `http://localhost:4000`.) Hence, I guess that `config.js` is saved, if really is, somewhere else.
- a message (e.g. modal) about the outcome of submission (e.g. succeeded, failed, config file created, etc.) is not displayed at all.
 
> Maybe that's what you are looking for?

IMHO, `mdstart /config` is a good quick way to start; but `-C, --config` is also a very common and "tested" way. Using the latter one, the config file could also lie in another place (subdirectory) with other config files than root directory. In fact, I cannot recall any other case/app where I need(ed) to write `app /config`.
